docs/libcurl/opts/CURLOPT_SSL_CTX_FUNCTION.3

```diff
@@ -61,6 +61,9 @@ necessary. For example, you can use this function to call library-specific
 callbacks to add additional validation code for certificates, and even to
 change the actual URI of an HTTPS request.
 
+For OpenSSL, error code \fISSL_ERROR_WANT_RETRY_VERIFY\fP is supported for
+custom certificate verification. (Added in 8.3.0)
```

Comment:
I don't understand.

The callback set with `CURLOPT_SSL_CTX_FUNCTION` is documented to use a `CURLcode` as return value. The man page example shows how it returns `CURLE_OK`.

How exactly is this "error code" set by the callback?

Reply:
Yes it could be misinterpreted. How about "For OpenSSL, asynchronous certificate verification via SSL_set_retry_verify is supported. (Added in 8.3.0)"

Reply:
Updated with your suggestion, thanks @jay!

OpenSSL's documentation of [SSL_set_retry_verify](https://www.openssl.org/docs/man3.1/man3/SSL_set_retry_verify.html) is a good entry point. It also points the reader to [SSL_CTX_set_cert_verify_callback](https://manpages.debian.org/testing/libssl-doc/SSL_CTX_set_cert_verify_callback.3ssl.en.html) which needs to be called from `CURLOPT_SSL_CTX_FUNCTION` to install the custom certificate verification function.